src/models/Accommodation.js

```diff
@@ -30,16 +30,16 @@ class Accommodation {
 
     @JsonProperty()
     @JsonType(String)
-    comments = "";
+    staffComments = "";
 
     @JsonConverter(new MomentSerializer())
     @JsonProperty()
     @JsonType(String)
     createdAt = undefined;
 
-    @JsonProperty()
-    @JsonType(String)
-    description = "";
+    // @JsonProperty()
+    // @JsonType(String)
+    // description = "";
```

Comment:
Please don't remove it! Unless this is replaced by `staffComments`? I didn't look at the recent API version yet, but anyway - we need both